tests/turbine_unit_test.py

```diff
@@ -304,62 +304,66 @@ def test_ct():
 
 
 def test_power():
-    N_TURBINES = 4
```

Comment:
Using multiple turbines tests an aspect of the vectorization. Could you maintain a test with multiple turbines?